loader/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("VulnerableLibrariesLocal")
```

Comment:
I wonder what library is supposedly vulnerable?

Reply:
It was the bundled Gson and Guava from Fabric's Mixin